app/controllers/histories_controller.rb

```diff
@@ -8,9 +8,45 @@ class HistoriesController < ApplicationController
   def create
     redirect_to root_url unless current_user.can_create_subject_history?
 
-    History.create!(patient_id: params.permit(:patient_id)[:patient_id],
+    history = History.new(patient_id: params.permit(:patient_id)[:patient_id],
```

Comment:
some of these params (especially `patient_id` and maybe `created_by` and `comment`) should be required (also applies to the other methods as well), @holmesie which params do you think we should require?

Reply:
started requiring some params such as `patient_id` and history `id` here: https://github.com/SaraAlert/SaraAlert/pull/793/commits/c8f1f62444e06ae8640e77e29be166045165d6dd#diff-c8c8197061d8db322cb11806e49c59c00413cfb234997e88adfe4e939477b69aR11

would like to know if we should require other ones as well

Reply:
I think this is fine since those are the only ones we use